hw5/index.html

```diff
@@ -0,0 +1,273 @@
+      </section>
+
+      <section class="journey container">
+        <div class="journey__content">
+          <div class="journey__wrapper">
+            <p class="journey__description--wrapper">Start your journey!</p>
+            <h1 class="journey__title">How to Get My Room?</h1>
+            <p class="journey__description--wrapper">
+              You can contact us by phone or email us. Easily tap the contact
+              button below and it will take you to our contact point
+            </p>
+            <button class="journey__button button">Contact us</button>
+            <button class="journey__button button--journey--ghost">
+              Explore more room
+            </button>
+          </div>
+
+          <div class="journey__choice">
+            <div class="journey__choose">
+              <h2 class="choice__title">1. Choose a room and date</h2>
+            </div>
+
+            <div class="journey__check">
+              <h2 class="choice__title">2. Check for room availability</h2>
+              <p class="journey__description">
+                Contact us via phone or email, ask for availability of you
+                choice of room. We might offer you something exciting.
+              </p>
+            </div>
+
+            <div class="journey__enjoy">
+              <h2 class="choice__title">3. Enjoy your room</h2>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="review container">
+        <div class="review__feedback">
+          <h1 class="review__title">Hear From Our Happy Customers</h1>
+          <img
+            class="review__image"
+            src="./images/customer.png"
+            alt="Kirana Dunham"
+          />
+          <p class="review__description">
+            “Great service, great food, great people. The scenery is also
+            beautiful, you can do so much activity even with your famility.
+            Worth every penny! you should come and see for your self. 5 out of
+            5! the waffle is great!”
+          </p>
+          <p class="review__customer">Kirana Dunham</p>
+          <div class="review__ponts">
+            <h1 class="review__circle-small"></h1>
+            <h1 class="review__circle-big"></h1>
+            <h1 class="review__circle-small"></h1>
+            <h1 class="review__circle-small"></h1>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="information container">
+      <div class="information__general">
+        <div class="right">
+          <h1 class="information__title">Cootels</h1>
+          <p class="information__description">Your Best Private</p>
+          <p class="information__description">Hideway From Crowd.</p>
+          <p class="information__description">Back to Nature.</p>
+          <img
+            class="imformation__image"
+            src="./images/Twitter.png"
+            alt="twitter"
+          />
+          <img
+            class="imformation__image"

```

Comment:
```suggestion
            class="information__image"

```
орфография это очень важно